.mergify.yml

```diff
@@ -1,21 +1,44 @@
-pull_request_rules:
+queue_rules:
+  - name: default
+    conditions:
+      # Conditions to get out of the queue (= merged)
+      - check-success=Check / Code Style
+      - check-success=Check / Binary Compatibility
+      - check-success=Check / Docs
+      - check-success=Check / Tests (Scala 2.12.15 & JDK 11)
+      - check-success=Check / Tests (Scala 2.13.7 & JDK 11)
+      - check-success=Check / Tests (Scala 3.0.2 & JDK 11)
+      - check-success=Check / Tests (Scala 2.12.15 & JDK 8)
+      - check-success=Check / Tests (Scala 2.13.7 & JDK 8)
+      - check-success=Check / Tests (Scala 3.0.2 & JDK 8)
```

Comment:
Hmm.. Would probably be nice if we could catch all with e.g. `check-success=Check / *`

Reply:
@mkurz Unfortunately not 😞 
https://docs.mergify.com/conditions/#validating-all-status-checks

Reply:
:cry: